frontend/src/components/identities/AccountForm.vue

```diff
@@ -27,7 +27,7 @@
         </v-row>
       </v-expansion-panel-header>
       <v-expansion-panel-content>
-        <account-role-form ref="roleForm" v-model="editedAccount.role" :account="editedAccount" />
+        <account-role-form ref="roleForm" v-model="editedAccount.role" :account="editedAccount" :authUserRole="authUser.role"/>
```

Comment:
Since there is only one authUser and it comes from a store, maybe you could use the getter directly inside the account-role-form component?